lib/index.ts

```diff
@@ -327,23 +327,43 @@ export function main() {
     return;
   }
 
-  for (const filePath of files) {
-    // Read and parse the source file.
-    const ext = path.extname(filePath);
-    const dir = outDir || path.dirname(filePath);
-    const outPath = path.join(dir, path.basename(filePath, ext) + suffix + (options.format === "ts" ? ".ts" : ".js"));
-
-    if (changedOnly && !needsUpdate(filePath, outPath)) {
+  if (
+    files.length === 1
+    && files[0].match(/\*/)
+  ) {
+    if (verbose) {
+      console.log(`Globbing ${files[0]}`);
+    }
+    var glob = require("glob")
+    glob(files[0], function (error: any, globFiles: string[]) {
```

Comment:
Thanks for reverting the whitespace changes, now I can see the actual diff.

You can simplify quite a bit. With glob, there is no need to check whether to apply it: a path with no patterns will be returned as is. So you can do something like this:
```
var glob = require('glob');  // on top of file

const globFiles = [].concat(...files.map(p => glob.sync(p)));
```
And then use old code, only replacing `files` with `globFiles`.

The build needs to pass. The failure seems due to your omitting changedOnly/needsUpdate checks. You can test yourself by running `npm test`.